blob/fileblob/fileblob.go

```diff
@@ -176,6 +207,9 @@ type Options struct {
 	// If true, create the directory backing the Bucket if it does not exist
 	// (using os.MkdirAll).
 	CreateDir bool
+
+	// Refers to the strategy for how to deal with metadata, such as blob.Attributes.
```

Comment:
Any reason why you changed this to a string from a bool Option? Do you anticipate other values, or ....?

Reply:
Yes, I've this for *xattr* (“extended attributes”) implemented, but need to get to a better understanding of its limits, and this PR as base to actually extend both `writer*`, before filing a draft.